test/unit/api-client/api-client.submit.spec.ts

```diff
@@ -36,13 +36,38 @@ describe('api-client job submission', () => {
             expect(job).toEqual(jobDetails);
         });
 
+        it('submit job with media url with null options', async () => {
+            const mockHandler = ApiRequestHandler.mock.instances[0];
+            mockHandler.makeApiRequest.mockResolvedValue(jobDetails);
+            const options = {
+                metadata: null,
+                callback_url: null,
+                custom_vocabularies: null,
+                skip_punctuation: null,
+                skip_diarization: null,
+                speaker_channels_count: null,
+                filter_profanity: null
+            };
+
+            const job = await sut.submitJobUrl(mediaUrl, null);
+
+            expect(mockHandler.makeApiRequest).toBeCalledWith('post', '/jobs',
+                { 'Content-Type': 'application/json' }, 'json', { media_url: mediaUrl });
```

Comment:
Not sure if we structure java code the same as C#, but maybe have the ); on a new line & each parameter for .toBeCalledWith on a new line as well

Reply:
It passes the lint check script and seems to be present in other places as well so I think it's fine?